src/home_robot_sim/home_robot_sim/env/habitat_objectnav_env/constants.py

```diff
@@ -520,40 +524,119 @@ def num_sem_categories(self):
     "can_opener": 24,
     "can": 25,
     # receptacles ->
-    "console_table": 26,
-    "sink_cabinet": 27,
-    "shoe_rack": 28,
-    "buffet": 29,
-    "footstool": 30,
-    "toilet": 31,
-    "tv_stand": 32,
-    "chest_of_drawers": 33,
-    "l-shaped_couch": 34,
-    "stool": 35,
-    "rocking_chair": 36,
-    "chair": 37,
-    "base_cabinet": 38,
-    "bar_stool": 39,
-    "cabinet": 40,
-    "bathtub": 41,
-    "sofa": 42,
-    "table": 43,
-    "armchair": 44,
-    "kitchen_island": 45,
-    "end_table": 46,
-    "washer": 47,
-    "swivel_chair": 48,
-    "kitchen_cabinet": 49,
-    "trunk": 50,
-    "nightstand": 51,
-    "serving_cart": 52,
-    "dining_table": 53,
-    "bench": 54,
-    "beanbag_chair": 55,
-    "coffee_table": 56,
-    "air_hockey_table": 57,
-    "desk": 58,
-    "storage_bench": 59,
+    # "console_table": 26,
```

Comment:
delete?

Reply:
Removed this altogether since we don't need the hardcoded categories anymore, we're loading them from an external file.